### PR TITLE
Tienle/custmaa

### PR DESCRIPTION
**Pull request by tle15**
Added custom MAA endpoint and ohttp server dockerfile.

### CODE REVIEW

File: cgpuvm-attest/src/lib.rs [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 
 #[link(name = "azguestattestation")]
 extern {
-    fn get_attestation_token(app_data: *const u8, pcr_sel: u32, jwt: *mut u8,  jwt_len: *mut size_t) -> c_int;
+    fn get_attestation_token(app_data: *const u8, pcr_sel: u32, jwt: *mut u8,  jwt_len: *mut size_t, endpoint_url: *const u8) -> c_int;
 }
 
 pub fn attest(data: &[u8], pcrs: u32) -> Option<Vec<u8>> {
     unsafe {
+        let url_ptr: *const u8 = MAA_ENDPOINT_URL.as_ptr();
```
Comment (kapilvgit) [inline cgpuvm-attest/src/lib.rs:12]: can we add this as a parameter to the server?

File: ohttp-server/libazguestattestation.so.1.0.5
Comment (kapilvgit) [inline ohttp-server/libazguestattestation.so.1.0.5:1]: I dont think we should checkin the binary. Either we should take a dependency on a debian package or build the guest attestation library ourselves.

File: cgpuvm-attest/src/lib.rs (hunk 1 above)
Comment (tle15) [reply cgpuvm-attest/src/lib.rs]: @kapilvgit : I've updated it.

File: ohttp-server/libazguestattestation.so.1.0.5
Comment (tle15) [reply ohttp-server/libazguestattestation.so.1.0.5:1]: This is still only for POC. We are still waiting for the official CVM guest attestation lib update, once it is done, we will be able to get the binary from https://packages.microsoft.com/repos/azurecore/pool/main/a/azguestattestation1/.

**Review by kapilvgit — APPROVED** [review]

**Pull request merged**